### PR TITLE
feat: many timers

### PR DESCRIPTION
**Pull request by cpvalente**
This PR adds the support for extra timers in Ontime.
So far, we dont have support for the extra timers to cause any other actions in the application

Tasks:
- [x] Create simple timer abstraction
- [x] Create API endpoints to control the timer
- [x] Create necessary UI
- [x] Add wanted amount of timers

### CODE REVIEW

File: apps/server/src/classes/simple-timer/SimpleTimer.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+   * Sets the duration of the timer
+   * @param time - time in milliseconds
+   */
+  public setTime(time: number): SimpleTimerState {
+    this.state.duration = time;
+    this.state.current = time;
+    return this.state;
+  }
+
+  public play(timeNow: number): SimpleTimerState {
```
Comment (cpvalente) [inline apps/server/src/classes/simple-timer/SimpleTimer.ts:32]: should probably be called start like the other timer

File: packages/types/src/definitions/runtime/Extratimer.type.ts [hunk 2]
```diff
@@ -1,7 +1,9 @@
 export type SimplePlayback = 'play' | 'pause' | 'stop';
+export type SimpleDirection = 'up' | 'down';
```
Comment (cpvalente) [inline packages/types/src/definitions/runtime/Extratimer.type.ts:2]: Suggestion: how about something more semantic of timers, like 
`type: 'count-down' | 'count-up'`

It achieves the same but has a similar wording to the other timers, which may make it less confusing for us

File: apps/client/src/common/components/input/time-input/TimeInput.tsx [hunk 3]
```diff
@@ -7,10 +7,13 @@ import { forgivingStringToMillis } from '../../../utils/dateConfig';
 import { TimeEntryField } from '../../../utils/timesManager';
 
 import style from './TimeInput.module.scss';
+
+type OntimeTimeEntry = TimeEntryField | 'extraTimer'
```
Comment (alex-Arc) [inline apps/client/src/common/components/input/time-input/TimeInput.tsx:11]: @cpvalente tcs seams to be complaining about this, I'm unsure how to fix it

**Comment by alex-Arc** [comment]
@cpvalente I think this is a good start for now

**Pull request merged**